jui-core/index.d.ts

```diff
@@ -0,0 +1,819 @@
+// Type definitions for jui-core 2.0
+// Project: https://github.com/juijs/jui-core
+// Definitions by: JinHo Park <https://github.com/easylogic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery"/>
+
+declare var jui: JuiStatic;
+
+interface UtilBase {
+    /**
+     * @property browser check browser agent
+     * @property {Boolean} browser.webkit  Webkit 브라우저 체크
+     * @property {Boolean} browser.mozilla  Mozilla 브라우저 체크
+     * @property {Boolean} browser.msie  IE 브라우저 체크 */
+    browser: {
+        webkit: boolean,
+        mozilla: boolean,
+        msie: boolean
+    };
+
+    /**
+     * @property {Boolean} isTouch
+     * check touch device
+     */
+    isTouch: boolean;
+
+    /**
+     * @method inherit
+     *
+     * 프로토타입 기반의 상속 제공
+     *
+     * @param {Function} ctor base Class
+     * @param {Function} superCtor super Class
+     */
+    inherit (ctor: () => void, superCtor: () => void): void;
```

Comment:
Documentation says `base Class`, so you should probably use `Function` as the type of parameters. (And above this have `tslint:disable-next-line:callable-types` or else the linter will complain.)